BeeSwiftIntents/AddDataIntentHandler.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AddDataIntentHandler.swift
+//  BeeSwift
+//
+//  Created by Andrew Brett on 3/28/21.
+//  Copyright © 2021 APB. All rights reserved.
+//
+
+import Foundation
+import Intents
+
+class AddDataIntentHandler: INExtension, AddDataIntentHandling {
+    func resolveValue(for intent: AddDataIntent, with completion: @escaping (AddDataValueResolutionResult) -> Void) {
+        print("foo")
```

Comment:
The use of a logging framework along with helpful logging messages, for use during development, debugging, and troubleshooting sure do make sense.